node/src/components/consensus/protocols/zug.rs

```diff
@@ -1516,6 +1547,18 @@ impl<C: Context + 'static> Zug<C> {
                 "could not create a WAL using this file"
             ),
         }
+        /*
+        if let Some((id, round)) = self.rounds.last_key_value() {
+            //proposals
+            //echoes
+            //votes
+            round
+        }
+        */
+
+        /*
+        find whatever the last round was, get the content for this validator and resend the messages
+         */
```

Comment:
cut this and apply it the ticket created for the future work to add the capability